perfmetrics/scripts/gsheet/gsheet_test.py

```diff
@@ -16,6 +17,147 @@ class MockCredentials:
   def authorize(self, request):
     return request
 
+def call_write_to_google_sheet_function(self):
```

Comment:
For reviewer: This is the existing logic from [master](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/gsheet/gsheet_test.py#L30). 

returned sheets_service_mock and calls to compare if sheet is updated or not in both the cases(env var is set or not set).